src/transform/ThemeTransform.js

```diff
@@ -2,11 +2,11 @@ import './ThemeTransform.css'
 import ElementUtilities from './ElementUtilities'
 import Polyfill from './Polyfill'
 
-// Elements marked with either of these classes indicate certain ancestry constraints that are
+// Elements marked with these classes indicate certain ancestry constraints that are
 // difficult to describe as CSS selectors.
 const CONSTRAINT = {
   IMAGE_NO_BACKGROUND: 'pagelib-theme-image-no-background',
-  IMAGE_NONTABULAR: 'pagelib-theme-image-nontabular'
+  DIV_DO_NOT_APPLY_BASELINE: 'pagelib-theme-div-do-not-apply-baseline'
```

Comment:
This is all really good information, but the reason I needed to switch to a `pagelib-theme-div-do-not-apply-baseline` approach is CSS doesn't seem to handle multiple `:not()`'s. If you look at the theming CSS for images specifically it was already very difficult to reason about, and w/o being able to specify additional `:not()` classes it couldn't be extended.